### ISSUE TITLE
Validation set and test set

### ISSUE THREAD

**Issue by cloudyliy** [TRKuan/IDRiD#5]
train.py:
data_train_dir = './data/sub1/train'
data_val_dir = './data/sub1/val'

test.py:
data_dir = './data/sub1/val'

Are the validation set and test set the same in the code?

**Comment by TRKuan** [comment]
`test.py` is for running predictions and get some understanding of the results. Just set `data_dir` to whatever dataset you want to evaluate.
